codejail/safe_exec.py

```diff
@@ -4,38 +4,20 @@
 import os.path
 import shutil
 import sys
-import textwrap
 
-try:
-    import simplejson as json
-except ImportError:
-    import json
-
-from codejail import jail_code
-from codejail.util import temp_directory, change_directory
+from .exceptions import SafeExecException
+from . import jail_code
+from .util import temp_directory, change_directory, json_safe
 
 log = logging.getLogger("codejail")
 
 
 # Flags to let developers temporarily change some behavior in this file.
 
-# Set this to True to log all the code and globals being executed.
-LOG_ALL_CODE = False
 # Set this to True to use the unsafe code, so that you can debug it.
 ALWAYS_BE_UNSAFE = False
 
 
-class SafeExecException(Exception):
-    """
-    Python code running in the sandbox has failed.
-
-    The message will be the stdout of the sandboxed process, which will usually
-    contain the original exception message.
-
-    """
-    pass
-
-
 def safe_exec(code, globals_dict, files=None, python_path=None, slug=None,
```

Comment:
The duplication of this docstring in jail_code.py is unfortunate.  You must have had a reason to move the bulk of this function into jail_code, but I'm not seeing it through my review telescope.  There's not much left in this file.